### ISSUE TITLE
REQUEST: New Approver status for chinikins

### ISSUE THREAD

**Issue by Chinikins** [spinnaker/governance#160]
### Github Username

@chinikins

### Requirements

- [x] Belongs to an organization with significant stake in project
- [x] Known active participant in the community
- [x] Has submitted at least 5 PRs of significant scope that are merged
- [x] Is already fulfilling the Reviewer role
- [x] Has reviewed 5 PRs of significant scope
- [x] Sponsored by 2 Approvers

### Sponsors

- @kevinawoo
- @dnilasor
- @ezimanyi

### List of qualifications for this position

- Name of organization(s) with significant stake in project: Cloudera
- Links to public Spinnaker PRs: https://github.com/pulls?q=is%3Apr+sort%3Aupdated-desc+author%3AChinikins+

I'm following up on https://github.com/spinnaker/governance/issues/156 and have come back after more reviews and PR submissions to continue on a track to become a maintainer for the Spin CLI and Sponnet projects!

**Comment by kevinawoo** [comment]
I've worked with @Chinikins, and she's been super helpful a bunch of `spin`'s issues.

We've worked together on the issue https://github.com/spinnaker/spinnaker/issues/5945 where I "fixed" the spin cli with PR https://github.com/spinnaker/spin/pull/289 but this caused a regression. This required a fix and also prompting a [backport to 1.21.x](https://github.com/spinnaker/spin/pull/290) to address the regression.

She's also been working with the community on https://github.com/spinnaker/spin/pull/269, which was closed in favor of https://github.com/spinnaker/spin/pull/262. As of writing, the issue still seems to be open, and she's following up with the community.

She's also been involved with reviewing and getting this PR https://github.com/spinnaker/spin/pull/257 merged, not commenting directly, but through slack and connecting it with the issue https://github.com/spinnaker/spinnaker/issues/5900.

In summary, she's been really helpful in helping troubleshoot issues, reviewing and testing PRs, communicating with the community, and doing a lot of the discovery work that I think is involved with approver-ship. On top of the [Eric's comments](https://github.com/spinnaker/governance/issues/156#issuecomment-665360379) from the previous request, she's got my sponsorship! 👍 💯 🅰️➕

**Comment by ezimanyi** [comment]
+1 both as TOC vote and as sponsor!

**Comment by ethanfrogers** [comment]
TOC +1

**Comment by robzienert** [comment]
TOC +1

**Comment by dnilasor** [comment]
+1 as a Sponsor, @Chinikins has demonstrated a highly collaborative and professional approach to contributing and assisting others. She represents the user community as part of the Spinnaker team at Cloudera and brings valuable knowledge of operating Spinnaker at scale. 

Giant thanks to @ezimanyi @kevinawoo @ethanfrogers for getting Chini on her way so swiftly! @Chinikins , go forth and maintain the Spin CLI, lol.

**Comment by ajordens** [comment]
TOC +1 but I'd like to see the approver status limited to `spin` and `sponnet` repositories _(I believe that's the intention here anyhow)_.
